app/services/foreman_rh_cloud/template_renderer_helper.rb

```diff
@@ -20,11 +20,12 @@ def remediations_playbook(hit_remediation_pairs)
     end
 
     apipie :method, 'Returns a Red Hat remediation playbook compiled on console.redhat.com' do
-      required :remediation_path, String, desc: ''
+      required :playbook_url, String, desc: 'URL of the playbook on console.redhat.com'
+      required :organization_id, Integer, desc: 'Id of the organization that owns the playbook'
```

Comment:
@adamruzicka Can you take a look at this part? Maybe there is a more elegant way to pass the organization

Reply:
If I'm reading things right then the job is kicked off with `Organization.current` being set to the right org. You should be able to leverage it and just use the current organization, although it is a lot less explicit

Reply:
Well, let's keep the caller context and action parameters separate. I think it would be better to be explicit in this case.

Reply:
Agreed